packages/components/psammead-most-read/README.md

```diff
@@ -117,7 +117,7 @@ import {
 | listIndex | number | yes | N/A | `1` |
 | numberOfItems | number | yes | N/A | `10` |
 | dir | string | no | `"ltr"` | `"ltr"` |
-| maxTwoColumns | bool | no | `false` | `true`|
+| columnLayout | string | no | 'multiColumn' | `multiColumn`
```

Comment:
Can you put something different in "Example" column for consistency please (e.g. `twoColumn`)? Just because values in "Default" and "Example" are usually different from what I've seen